src/widgets/grafana/component.jsx

```diff
@@ -10,12 +10,23 @@ export default function Component({ service }) {
   const { widget } = service;
   const { data: statsData, error: statsError } = useWidgetAPI(widget, "stats");
   const { data: alertsData, error: alertsError } = useWidgetAPI(widget, "alerts");
+  const { data: alertmanagerData, error: alertmanagerError } = useWidgetAPI(widget, "alertmanager");
 
-  if (statsError || alertsError) {
+  let alertsInt = 0;
+
+  if ((alertsError && alertsError.message === "HTTP Error") || !alertsData || alertsData.length === 0) {
```

Comment:
```suggestion
  if (alertsError || !alertsData || alertsData.length === 0) {
```

It should just be enough to check for the error. If the entire Grafana instance is down then everything should error out, so this wont be a problem and the later conditional will catch it